spec/dummy/db/schema.rb

```diff
@@ -173,6 +173,18 @@
     t.string   "base_url",                                     null: false
   end
 
+  create_table "atmosphere_migration_jobs", force: true do |t|
+    t.integer  "appliance_type_id"
+    t.integer  "virtual_machine_template_id"
+    t.integer  "compute_site_source_id"
+    t.integer  "compute_site_destination_id"
+    t.text     "status"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  add_index "atmosphere_migration_jobs", ["appliance_type_id", "virtual_machine_template_id", "compute_site_source_id", "compute_site_destination_id"], name: "atmo_mj_ix", unique: true, using: :btree
```

Comment:
Line is too long. [199/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.